docs/content/docs/reference/types.md

```diff
@@ -20,18 +20,26 @@ is represented by `time.Time`, and so on.
 
 User-defined types referenced by a verb will be automatically exported as FTL types.
 
-| FTL             | Go    | Kotlin | Java |
-| :-------------- | :---- | :----- | :--- |
-| `Int`             | `int`    | `Long`      | `Long`    |
-| `Float`           | `float64`    | `Double`      | `Double`    |
-| `String`          | `string`    | `String`      | `String`    |
-| `Bytes`           | `[]byte`    | `ByteArray`      | `ByteArray`    |
-| `Bool`            | `bool`    | `Boolean`      | `Boolean`    |
-| `Time`            | `time.Time`    | `ZonedDateTime`      | `ZonedDateTime️ `   |
-| `Any`             | [external](../externaltypes)    | [external](../externaltypes)      | [external](../externaltypes)    |
-| `Unit`            |     |       |     |
-| `Map<K,V>`        | `map[K]V`    | `Map<K,V>`      | `Map<K,V>`    |
-| `Array<T>`        | `[]T`    | `List<T>`      | `List<T>`    |
+| FTL                | Go                           | Kotlin                       | Java                         |
+|:-------------------|:-----------------------------|:-----------------------------|:-----------------------------|
+| `Int`              | `int`                        | `Long`                       | `long`                       |
+| `optional[Int]`    | `ftl.Optional[int]`          | `Long?`                      | `Long`                       |
```

Comment:
FYI FTL represents optional with a `?`, exactly like Kotlin.

I think given that 99% of these optionals are uniform, except for Java, maybe it's better to just call out how Java is different in the "Optional" section?

Reply:
Maybe a new column instead? `Java (optional)`

Reply:
I have updated the table, and added some additional notes.